doc/onboarding-extras.md

```diff
@@ -27,6 +27,7 @@
 | `test/*` | @nodejs/testing |
 | `tools/eslint`, `.eslintrc` | @not-an-aardvark, @silverwind, @trott |
 | async_hooks | @nodejs/diagnostics |
+| performance | @nodejs/performance |
```

Comment:
It already has @nodejs/benhmarking and mscdex is not part of that team but is interested in the core benchmarks